Exercise_A/index.html

```diff
@@ -10,6 +10,8 @@
   </head>
   <body>
     <!-- CODE HERE -->
+    <p>this is index page</p>
+    <a href="about.html">About Link</a>
 
     <script src="" async defer></script>
   </body>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!